src/compiled-grammar/index.ts

```diff
@@ -47,7 +47,9 @@ export interface CompiledRule {
      * `allowedSyntaxKinds(choice())` or,
      * `allowedSyntaxKinds(SyntaxKind.SomeChoiceRule)`
      */
-     allowedSyntaxKinds? : string[] | undefined;
+    allowedSyntaxKinds? : string[] | undefined;
+
+    postParse? : string | undefined;
```

Comment:
Considering deleting this since it never ended up getting used.
It complicates stuff and didn't really produce any solid benefit.